src/script/team/TeamState.ts

```diff
@@ -36,6 +36,8 @@ export class TeamState {
   public readonly supportsLegalHold: ko.Observable<boolean>;
   public readonly teamName: ko.PureComputed<string>;
   public readonly teamFeatures: ko.Observable<FeatureList>;
+  public readonly isConferenceCallingEnabled: ko.PureComputed<boolean>;
+  public readonly isVideoCallingEnabled: ko.PureComputed<boolean>;
```

Comment:
where are these two used?

Reply:
They are not used yet. Basically this is an except of a bigger feature and this is the ground work for it. I just wanted to split it a bit in order to have readable PRs :)